Task2/Index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="content.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <title>Task2</title>
+</head>
+<body>
+<div class="list">
+    <h1>Chapters</h1>
+    <ul>
+        <li><a href="#a1">What is Lorem ipsum</a></li>
+        <li><a href="#a2">Why do we us it</a></li>
+        <li><a href="#a3">Where does it come from?</a></li>
+        <li><a href="#a4">Where can I get some?</a></li>
+        <li><a href="#a5">There standards</a></li>
+        <li><a href="#a6">Section</a></li>
+        <li><a href="#a7">1914</a></li>
+        <li><a href="#a8">Section</a></li>
+        <li><a href="#a10">Donate</a></li>
+    </ul>
+</div>
+<div class="a0"><a href="https://www.google.com" target="_blank">GOOGLE</a></div>
```

Comment:
In my opinion the link "GOOGLE" it's a part of navigation, so it should be one of `<li>`.